spec/markup_helper_spec.rb

```diff
@@ -6,10 +6,7 @@
 
   it "can tokenize liquid source" do
     source = 'liquid'
-    text_stream = double('text_stream')
-    allow(TextStream).to receive(:new).with(source, mode: :liquid).and_return(text_stream)
-    allow(Scanner).to receive(:new).with(text_stream).and_return(t_ident(:x))
-    expect(tokenize(source).to_a).to eq(t_ident(:x))
+    expect(tokenize(source).to_a).to eq([t_text('liquid'), t_eos])
```

Comment:
@sgnr I was trying to find where we actually parse markup (I haven't found a place) but I stumbled on this overmocked test that actually does nothing and didn't catch an interface change.